Godeps/Godeps.json

```diff
@@ -3123,6 +3123,10 @@
 			"Rev": "ceefd2213ed29504fff30155163c8f59827734f3"
 		},
 		{
+			"ImportPath": "golang.org/x/time/rate",
```

Comment:
It was used in v1.5.0-beta.2, but reverted and switched back to juju/ratelimit. I just haven't adjusted Godeps.json to remove the ref to x/time/rate. Will do.